Setup/Product.wxs

```diff
@@ -1,393 +1,729 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:netfx="http://schemas.microsoft.com/wix/NetFxExtension"
     xmlns:VSExtension="http://schemas.microsoft.com/wix/VSExtension">
-  <?include Config.wxi?>
-  <Product Id="*" UpgradeCode="$(var.UpgradeCode)" Name="$(var.ProductName) $(var.Version)" Manufacturer="$(var.Manufacturer)" Version="$(var.NumericVersion)" Language="1033">
-    <Package InstallerVersion="200" Compressed="yes" Description="$(var.ProductName)" />
-    <Media Id="1" Cabinet="media1.cab" EmbedCab="yes" />
-    <Property Id="ALLUSERS" Value="1" />
-    <PropertyRef Id="WIX_IS_NETFRAMEWORK_461_OR_LATER_INSTALLED" />
-    <Condition Message=".NET Framework 4.6.1 must be installed prior to installation of Git Extensions.">
-        Installed OR WIX_IS_NETFRAMEWORK_461_OR_LATER_INSTALLED
-    </Condition>
-    <Icon Id="gitextensions.ico" SourceFile="..\Logo\git-extensions-logo.ico" />
-    <?include AddRemove.wxi?>
-    <?include EnableUpgrades.wxi?>
-    <UIRef Id="WixUI_GitExtensions" />
-    <Property Id="WIXUI_INSTALLDIR" Value="INSTALLDIR" />
-    <Property Id="SSHCLIENT" Value="PuTTY" Secure="yes" />
-    <WixVariable Id="WixUIDialogBmp" Value="dialog.bmp" />
-    <WixVariable Id="WixUIBannerBmp" Value="banner.bmp" />
-    <WixVariable Id="WixUISupportPerUser" Value="1" Overridable="yes" />
-    <WixVariable Id="WixUISupportPerMachine" Value="1" Overridable="yes" />
-    <Directory Id="TARGETDIR" Name="SourceDir">
-      <Directory Id="ProgramFilesFolder">
-        <Directory Id="INSTALLDIR" Name="$(var.InstallName)">
-          <Directory Id="PuttyDir" Name="PuTTY" />
-          <Directory Id="PluginsDir" Name="Plugins" />
-          <Directory Id="InstallerDir" Name="Installer" />
-          <Directory Id="DictionariesDir" Name="Dictionaries" />
-          <Directory Id="TranslationsDir" Name="Translation" />
-          <Directory Id="DiffScriptsDir" Name="Diff-Scripts" />
-          <Directory Id="DirConEmu" Name="ConEmu" />
+    <?include Config.wxi?>
```

Comment:
❗️ Can you revert the mass formatting change to a 4-space indent, or perform the formatting as a separate pull request? The reformatting makes the change unreviewable.

Reply:
Nope. You can compare in GE by selecting first and my latest commits and turning off whitespace in the diff view.

Reply:
Also GitHub has diff settings button.

Reply:
![screenshot_20190108-083533](https://user-images.githubusercontent.com/3629489/50833798-696ad100-1320-11e9-8235-4ba52d41e276.png)